tests/30rooms/11leaving.pl

```diff
@@ -0,0 +1,272 @@
+multi_test "Setup a room, and havve the first user leave (SPEC-216)",
+
+    requires => [qw(
+        make_test_room change_room_powerlevels
+            do_request_json_for user more_users
+        can_create_room
+    )],
+
+    provides => [qw( departed_room_id )],
+
+    # User A creates a room.
+    # User A invites User B to the room.
+    # User B joins the room.
+    # User B will set the ("m.room.name", "") state of the room to {
+    #   "body": "N1. B's room name before A left"
+    # }
+    # User B will set the ("madeup.test.state", "") state of the room to {
+    #   "body": "S1. B's state before A left"
+    # }
+    # User B will send a message with body "M1. B's message before A left"
+    # User B will send a message with body "M2. B's message before A left"
+    # User A will leave the room.
+    # User B will set the ("m.room.name", "") state of the room to {
+    #   "body": "N2. B's room name after A left"
+    # }
+    # User B will set the ("madeup.test.state", "") state of the room to {
+    #   "body": "S2. B's state after A left"
+    # }
+    # User B will send a message with text "M3. B's message after A left"
+    #
+    do => sub {
+        my (
+            $make_test_room, $change_room_powerlevels, $do_request_json_for,
+            $user_a, $more_users
+        ) = @_;
+        my $user_b = $more_users->[1];
+
+        my $room_id;
+
+        $make_test_room->($user_a, $user_b)->then( sub {
+            ( $room_id ) = @_;
+
+            provide departed_room_id => $room_id;
+
+            $change_room_powerlevels->($user_a, $room_id, sub {
+                my ( $levels ) = @_;
+                $levels->{users}{ $user_b->user_id } = 50;
+            })
+        })->then( sub {
+            $do_request_json_for->( $user_b,
+                method => "PUT",
+                uri => "/rooms/$room_id/state/m.room.name",
```

Comment:
Is this up to date with latest 'master'? Specifically, commit a356cf91 would mean that all these URIs now need /api/v1/ prepending to it